app/views/configs/_config.html.erb

```diff
@@ -1,15 +1,20 @@
 <div class="row">
   <div class="col-md-6">
     <%= bootstrap_form_for config, inline: true do |f| %>
-      <%= f.text_area :options, label: "#{config.config_key.capitalize} options", value: config.config_value[:options].join(', '), help: 'Please separate with commas.' %>
-      <%= f.submit "Update options for #{config.config_key}", class: 'btn btn-primary' %>
+      <%= f.text_area :options, label: "#{config.config_key.to_s.humanize} options",
```

Comment:
I strongly recommend digging thru activesupport stuff for http://api.rubyonrails.org/classes/String.html and http://api.rubyonrails.org/classes/Array.html -- all sorts of cool stuff and it's kind of a neat look into the thought process behind a lot of rails ('wouldn't it be cool if...'). String stuff like humanize is really, really nice and I miss it a lot out of rails.